### DISCUSSION TITLE
How can uninstall?

### DISCUSSION THREAD

**Discussion by xarmadigi** [hrfee/jfa-go#144] [Q&A]
Hello, by mistake I have deleted the user (admin) that creates the users. And I do not catch the changes. How can I completely uninstall and reinstall it?

sudo apt-get purge jfa-go has not worked for me. Thanks

**Comment by hrfee** [comment]
Just delete ~/.config/jfa-go to re-do the setup, or instead you can edit the config file with a new admin user at ~/.config/jfa-go/config.ini

**Reply by xarmadigi** [reply]
Yes!!! That's all right thanks
